src/pages/index.tsx

```diff
@@ -103,13 +103,13 @@ const features = [
     description: (
       <>
         <p>
-          <b>Codegen.</b> Generate tests by recording your actions. Save them into any language.
+          <b><a href="/docs/codegen">Codegen.</a></b> Generate tests by recording your actions. Save them into any language.
```

Comment:
```suggestion
          <b><a href="docs/codegen">Codegen.</a></b> Generate tests by recording your actions. Save them into any language.
```

I think changing it to this should make it work.